### ISSUE TITLE
config files for released models

### ISSUE THREAD

**Issue by cogmeta** [talonvoice/speech#1]
can you please provide us config files for the published models here  (https://talonvoice.com/research/) so that we can train on more data. We will be happy to contribute back if we end up improving it. Thanks.

**Comment by lunixbochs** [comment]
Thanks for the interest! The published models are all using the upstream librispeech recipe.

Also, wav2letter bundles the config files inside the models. You can copy any of my models into `your_model_dir/001_model_last.bin` then run `Train continue your_model_dir --flagsfile flagsfile` where your `flagsfile` mostly just specifies your additional training and test sets and learning rate.

Any flagsfile like this one works as a base:
https://github.com/talonvoice/speech/blob/master/projects/tedlium/flagsfile

However, I don't think that's the optimal way to train wav2letter. I also have a better model in progress that I haven't posted yet.

If you want to contribute to training in an effective manner, we should have a longer chat about your goals and what you plan to do. In theory I'm the person outside facebook who's run the most experiments on training w2l so far, and there are things that do and don't work that are just kinda in my head as intuition right now. I'm available for a call sometime today, or perhaps you can join the Talon slack and message me there.

**Comment by cogmeta** [comment]
@lunixbochs  Thank you so much. we will definitely take a look at the flagsfile extracted from  the models.  We are in the process of getting our hands on 6000 hrs (possibly a lot more ) of transcribed audio data and we would like to start using the data. I will message on Talon slack channel.  Thanks again.
